spec/models/card_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+require 'spec_helper'
+
+describe Card do
+  card = Card.create(original_text: "hausE SOmE ДоМ", translated_text: "hauses")
+  card_bad = Card.new(original_text: "home", translated_text: "home", review_date: Date.today)
+
+  it 'check translation' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.